src/types/expansion.ts

```diff
@@ -6,7 +6,7 @@ export type ExpansionRule = {
   authoring_command_dict_id: number;
   authoring_mission_model_id: number;
   created_at: string;
-  description: string;
+  description?: string;
```

Comment:
Same with [this one](https://github.com/NASA-AMMOS/aerie/blob/develop/sequencing-server/sql/sequencing/tables/expansion_rule.sql#L17)